modules/navier_stokes/test/tests/finite_volume/ins/jeffery-hamel/wedge_dirichlet_vol_PA.i

```diff
@@ -0,0 +1,192 @@
+mu=.01
+rho=1
+
+# This input file tests whether we can converge to the semi-analytical
+# solution for flow in a 2D wedge.
+[GlobalParams]
+  velocity_interp_method = 'rc'
+  advected_interp_method = 'average'
+  rhie_chow_user_object = 'rc'
+[]
+
+[Mesh]
+  [file]
+    type = FileMeshGenerator
+    # file = wedge_4x6.e
+    file = wedge_8x12.e
+    # file = wedge_16x24.e
+    # file = wedge_32x48.e
+    # file = wedge_64x96.e
+  []
+  [./corner_node]
+    # Pin is on the centerline of the channel on the left-hand side of
+    # the domain at r=1.  If you change the domain, you will need to
+    # update this pin location for the pressure exact solution to
+    # work.
+    type = ExtraNodesetGenerator
+    new_boundary = pinned_node
+    coord = '1 0'
+    input = file
+  [../]
+[]
+
+[UserObjects]
+  [rc]
+    type = INSFVRhieChowInterpolator
+    u = vel_x
+    v = vel_y
+    pressure = pressure
+  []
+[]
+
+[Variables]
+  [vel_x]
+    type = INSFVVelocityVariable
+  []
+  [vel_y]
+    type = INSFVVelocityVariable
+  []
+  [pressure]
+    type = INSFVPressureVariable
+  []
+[]
+
+[AuxVariables]
+  [U]
+    order = CONSTANT
+    family = MONOMIAL
+    fv = true
+  []
+[]
+
+[AuxKernels]
+  [mag]
+    type = VectorMagnitudeAux
+    variable = U
+    x = vel_x
+    y = vel_y
+  []
+[]
+
```

Comment:
```suggestion
```
Paraview should give us a magnitude for free since you have the variable component naming `vel_x` and `vel_y`